preset/azurite/README.md

```diff
@@ -0,0 +1,73 @@
+# Gnomock Blobstorage
```

Comment:
I'm not sure what's the best name for this one.

In localstack, we call it "localstack", and there are internal services that it exposes. Even though the users look for s3 or sqs, the preset is still named localstack.

I think we should keep this convention, and since you already named the preset `azurite`, I believe the readme should also use the term "azurite".

The Example (this is great by the way!) on the other hand should keep the name "blog storage" in it, because it demonstrates a subset of azurite capabilities.

Do you agree?

Reply:
Agreed

Reply:
With the test i'm fine. 

Or do you refer to the internal/gnomockd/azurite_test.go?
In this one I would leave it as "TestAzurite" because it is the same in localstack (it only tests s3 but is named localstack)